spec/spec_helper.rb

```diff
@@ -51,50 +51,50 @@
   config.filter_sensitive_data(opensearch_uri)   { ENV['OPENSEARCH_DESCRIPTION_URL'] }  if ENV['OPENSEARCH_DESCRIPTION_URL']
   config.filter_sensitive_data(hybrid_bills_uri) { ENV['HYBRID_BILL_API_BASE_URL'] }    if ENV['HYBRID_BILL_API_BASE_URL']
 
-  # Dynamically filter n-triple data
-  config.before_record do |interaction|
-    should_ignore = ['_:node', '^^<http://www.w3.org/2001/XMLSchema#date>', '^^<http://www.w3.org/2001/XMLSchema#dateTime>', '^^<http://www.w3.org/2001/XMLSchema#integer>']
-
-    # Check if content type header exists and if it includes application/n-triples
-    if interaction.response.headers['Content-Type'] && interaction.response.headers['Content-Type'].include?('application/n-triples')
-      # Split our data by line
-      lines = interaction.response.body.split("\n")
-
-      # How many times have we seen a predicate?
-      predicate_occurrances = Hash.new(1)
-
-      # Iterate over each line, decide if we need to filter it.
-      lines.each do |line|
-        next if should_ignore.any? { |condition| line.include?(condition) }
-        next unless line.include?('"')
-
-        # require 'pry'; binding.pry
-        # Split on '> <' to get a Subject and Predicate+Object split
-        subject, predicate_and_object = line.split('> <')
-
-        # Get the actual object
-        predicate, object = predicate_and_object.split('> "')
-
-        # Get the last part of a predicate URI
-        predicate_type = predicate.split('/').last
-
-        # Get the number of times we've seen this predicate
-        occurrance = predicate_occurrances[predicate_type]
-        predicate_occurrances[predicate_type] = predicate_occurrances[predicate_type] + 1
-
-        # Try and build a new object value based on the predicate
-        new_object = "#{predicate_type} - #{occurrance}\""
-
-        # Replace the object value
-        index = object.index('"')
-
-        object[0..index] = new_object if index
-
-        new_line = "#{subject}> <#{predicate}> \"#{object}"
-        config.filter_sensitive_data(new_line) { line }
-      end
-    end
-  end
+  # # Dynamically filter n-triple data
```

Comment:
Should this still be commented out? Maybe this is a larger discussion about whether we need this any more

Reply:
I have uncommented it, but I think it might be reasonable to have a conversation about if it is still needed.